### PR TITLE
add Gunicorn worker class to suport uvloop

### PR DESCRIPTION
**Pull request by drgarcia1986**
```python
import muffin


app = muffin.Application('uvloop')


@app.register('/')
def hello(request):
    return 'Hello with {0.__class__}'.format(app.loop)
```
```bash
$ gunicorn app:app --worker-class muffin.worker.GunicornUVLoopWorker --bind localhost:8080
```

```bash
$ curl -i localhost:8080/
HTTP/1.1 200 OK
CONTENT-TYPE: text/html; charset=utf-8
CONTENT-LENGTH: 32
DATE: Wed, 20 Jul 2016 12:10:06 GMT
SERVER: Python/3.5 aiohttp/0.21.6

Hello with <class 'uvloop.Loop'>
```

### CODE REVIEW

**Comment by drgarcia1986** [comment]
The build of travis fail to download python3.5 
```
3.5 is not installed; attempting download

The command "curl -s -o python-3.5.tar.bz2 ${archive_url}" failed and exited with 18 during .
```

@klen you can restart build ?

**Comment by 0nkery** [comment]
Does it have some performance gains?

**Comment by drgarcia1986** [comment]
@0nkery we need to check but in pratice uvloop is more fast than built-in asyncio loop (and uvloop has your own Future write in C with Cython)

File: muffin/worker.py [hunk 1]
```diff
@@ -98,3 +101,13 @@ def make_handler(self, app, *args):
             secure_proxy_ssl_header=app.cfg.SECURE_PROXY_SSL_HEADER
         )
         return handler
+
+
+class GunicornUVLoopWorker(GunicornWorker):
+    """Work with asyncio application (using uvloop)"""
+
+    def init_process(self):
+        import uvloop
```
Comment (jairhenrique) [inline muffin/worker.py]: this needs to be in some requirements file, right?

Comment (drgarcia1986) [inline muffin/worker.py]: no, because `uvloop` is optional, you only use if you want :smile:

Comment (jairhenrique) [inline muffin/worker.py]: would be cool if i could install muffin like this `pip install muffin[uvloop]`

Comment (drgarcia1986) [inline muffin/worker.py]: I don't follow for now, I try to using same approach of aiohttp

Comment (drgarcia1986) [inline muffin/worker.py]: But is not a bad idea

**Comment by klen** [comment]
Thank you!

**Pull request merged**